features/junit_format.feature

```diff
@@ -538,6 +538,7 @@
       """
     And the file "junit/default.xml" should be a valid document according to "junit.xsd"
 
+  @php-version @php5.3 @php5.4
```

Comment:
@stof this test fails on php 7 locally (imo is expected to fail), so do I remove the tags or not?